modules/commons/src/main/java/org/apache/synapse/commons/json/JsonStreamBuilder.java

```diff
@@ -43,12 +43,15 @@ public OMElement processDocument(InputStream inputStream, String s,
 
         if (inputStream != null) {
             OMElement element = JsonUtil.newJsonPayload(messageContext, inputStream, false, false);
-            if (element != null) {
-                if (logger.isDebugEnabled()) {
-                    logger.debug("#processDocument. Built JSON payload from JSON stream. MessageID: " + messageContext.getMessageID());
-                }
-                return element;
-            }
+			if (element != null) {
+				if (logger.isDebugEnabled()) {
```

Comment:
I have a doubt here. Coz we need to return element if it is not null despite of enabling/disabling debug logs.